gremlin-scala/src/main/scala/gremlin/scala/DefaultsToAny.scala

```diff
@@ -0,0 +1,9 @@
+package gremlin.scala
+
+sealed class DefaultsToAny[A]
```

Comment:
In [Dotty](https://github.com/lampepfl/dotty) that would have been `def whatever[A = Any](...): A`
The same with when we wanted to do `def whatever(keyValues: (String | T, Any)*): A` that seems to work in Dotty